docs/node/web3.md

```diff
@@ -0,0 +1,201 @@
+
+### Hardware
+
+* CPU:
+  * Minimum: 2.0 GHz x86-64 CPU
+  * Recommended: 2.0 GHz x86-64 CPU with 2 cores/vCPUs
+* Memory:
+  * Minimum: 4 GB of ECC RAM
+  * Recommended: 8 GB of ECC RAM
+* Storage:
+  * Minimum: 300 GB of SSD or NVMe fast storage
+  * Recommended: 500 GB of SSD or NVMe fast storage
+
+:::info
+
+To put the figures above into perspective, the Web3 gateway for Emerald with
+PostgreSQL encountered **210 GB** of database growth in ~5 months between
+Nov 18, 2021 and Apr 11, 2022 (since the [Emerald Mainnet launch]).
+
+:::
+
+### PostgreSQL
+
+The Web3 gateway stores blockchain data in a
+[PostgreSQL](https://www.postgresql.org/) database version **13.3** or higher.
+Install it by following the instructions specific to your operating system. You
+will need to create a separate database for each ParaTime. We also suggest that
+you create and use a different username for connecting to the database for each
+ParaTime.
+
+### Oasis ParaTime Client Node
+
+The Web3 gateway requires a locally deployed ParaTime-enabled Oasis Node.
+First, follow the
+[Oasis ParaTime Client Node](run-your-node/paratime-client-node.mdx)
+guide on how to configure the Oasis client node with one or more ParaTimes.
+Make sure you check the network parameters page ([Mainnet], [Testnet]) and use
+the exact combination of the Oasis node/ParaTime versions!
+
+Next, we suggest that you slightly modify the ParaTime section of your Oasis
+node configuration. By default, the Emerald and Sapphire ParaTimes prevent
+you from executing *expensive* transactions such as simulating EVM calls
+without paying for gas. If your Oasis node instance will only be used by the
+Web3 gateway(s) and otherwise not exposed to the internet, we suggest that you
+enable expensive transactions by adding the following to your Oasis node
+config:
+
+```yaml title='config.yml'
+# ... sections not relevant are omitted ...
+runtime:
+  mode: client
+  paths:
+    - <emerald_bundle_path>
+    - <sapphire_bundle_path>
+
+  config:
+    # Use the following IDs:
+    # 000000000000000000000000000000000000000000000000e2eaa99fc008f87f # Emerald on Mainnet
+    # 00000000000000000000000000000000000000000000000072c8215e60d5bca7 # Emerald on Testnet
+    # 000000000000000000000000000000000000000000000000a6d1e3ebf60dff6c # Sapphire on Testnet
+    "<emerald_paratime_id>":
+      estimate_gas_by_simulating_contracts: true
+      allowed_queries:
+        - all_expensive: true
+    "<sapphire_paratime_id>":
+      estimate_gas_by_simulating_contracts: true
+      allowed_queries:
+        - all_expensive: true
+```
+
+## Download the Web3 Gateway
+
+Check the required version of the Web3 gateway based on which network you will
+be using it with ([Mainnet], [Testnet]). Next, download the Oasis-provided
+binaries from the [official github repository][github-releases].
+
+Alternatively, you can download the source release and compile it yourself.
+Consult the [README] file for more information.
```

Comment:
```suggestion
Consult the [README.md] file for more information.
```
Because you are referring to a file, it might make sense to specify the exact filename (or even relative path)?